### PR TITLE
Updates intake model to reflect the fixes in the application repo

### LINKED ISSUE #302: Update intake library

### PR DESCRIPTION
**Pull request by pitchinvasion**
Closes #302

### CODE REVIEW

File: spec/intake_spec.rb [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+    Timecop.return
   end
 
-  let(:precourse_start_date) do
-    Date.today + intake_offset - described_class::PRECOURSE_LENGTH
+  let(:future_intake) do
+    {
+      "start_date" => "12th October 2015",
+      "end_date" => "15th January 2016"
+    }
```
Comment (sjmog) [inline spec/intake_spec.rb]: Is there a reason for using the old hash syntax? (Genuine question)

**Comment by sjmog** [comment]
:+1: :shipit:

File: spec/intake_spec.rb (hunk 1 above)
Comment (spike01) [inline spec/intake_spec.rb]: Keys are strings

**Comment by roidriscoll** [comment]
Bon. :shipit:

**Pull request merged**